src/chrome/content/rules/dlvr.it.xml

```diff
@@ -29,20 +23,13 @@
 		<test url="http://api.dlvrit.com/1/help" />
 	<target host="app.dlvrit.com" />
 	<target host="blog.dlvrit.com" />
-	<target host="dev.dlvrit.com" />
 	<target host="static.dlvrit.com" />
 		<test url="http://static.dlvrit.com/font-awesome/css/font-awesome.min.css" />
 	<target host="support.dlvrit.com" />
 
```

Comment:
```xml
<securecookie host=".+" name=".+" />
```

Reply:
No... do not mess with our site's cookies.  We very intentionally have portions of our site which are not secure and will not respond to https.  Modifying cookies will break things.

Reply:
@signe See https://www.eff.org/https-everywhere/faq#how-do-i-add-my-own-site-to-https-everywhere.

Reply:
What does that have to do anything?  It's irrelevant.

Our site already runs in https, without any assistance, and has for many years.  Someone else added this ruleset over a year ago and I'm fixing it because it was broken.

As I stated, portions of our site are intentionally http.  Attempts to force them to https will break them.   We do not _want_ them to be https, so I do not want a rule added which will intentionally break the usage of any cookies.

Reply:
Website owners should generally try to use HSTS preloading instead of creating HTTPS Everywhere rulesets.

Unencrypted HTTP is going away. Soon any unencrypted HTTP connection attempt will result in a security warning in all browsers. I recommend to upgrade your entire website to HTTPS and use HSTS preloading before this happens.

It is unlikely that this PR will be accepted unless you can demonstrate that upgrading the removed domains results in breakage.

If keeping a part of the website unencrypted is a hard requirement, I recommend using a different domain for unencrypted content, and preloading the encrypted domain.

Reply:
As I stated, _someone else_ created this ruleset some year or more ago.  All I have done is remove rules for hosts which no longer exist or which break functionality.

I would much rather there be no ruleset at all, as our site operates exactly as intended and we automatically upgrade all connections that are supposed to be upgraded.